kod/util/system.kod

```diff
@@ -474,6 +483,48 @@ messages:
       return;
    }
 
+   RetrieveServerConfigValues()
+   "Retrieves server config values needed for operation."
+   {
+      piMoveInterval = Send(self,@RetrieveServerConfigValue,#config_id=SERVER_CONFIG_MOVE_INTERVAL);
+      if piMoveInterval = $
+      {
+         piMoveInterval = MOVE_INTERVAL;
+      }
+
+      piMoveCountThreshold = Send(self,@RetrieveServerConfigValue,#config_id=SERVER_CONFIG_MOVE_COUNT_THRESHOLD);
+      if piMoveCountThreshold = $
+      {
+         piMoveCountThreshold = MOVE_COUNT_THRESHOLD;
+      }
+
+      return;
+   }
+
+   RetrieveServerConfigValue(config_id=$)
+   {
+      local config_value;
+
+      config_value = GetServerConfigValue(config_id);
+
+      if config_value = $
```

Comment:
Not sure I see the purpose of this function.  If the config_value is ```$```, it returns ```$```.  What does this function add over calling GetServerConfigValue directly?